nodejs/week1/warmup.js

```diff
@@ -0,0 +1,14 @@
+const argv = process.argv.slice(2);
+const sumArgv = argv.reduce((a, b) => Number(a) + Number(b));
+const average = sumArgv/ argv.length;
+
+//What should happen if the provided arguments are not numbers 
+if (isNaN(average)){
+ console.log("You have Characters Pls Type Only Number!")
+}
+
```

Comment:
else should be right where if block ends.
https://stackoverflow.com/a/4005623/4261654